native/tools/swig-config/assets.i

```diff
@@ -126,6 +126,7 @@
 //  4. 'Attribute Section' should be placed before 'Import Section' and 'Include Section'
 //
 %attribute(cc::Asset, ccstd::string&, _uuid, getUuid, setUuid);
+%attribute(cc::Asset, ccstd::string&, uuid, getUuid, setUuid);
```

Comment:
uuid only has a getter, no setter.

Reply:
Good finding, we need to remove setUuid at this line.

Reply:
OK，setUuid has been removed。